README.md

```diff
@@ -95,10 +95,16 @@ To run tests after setting up the linters, run the following command:
 
 👤 **Faiz Mohammad**
 
-- GitHub: [@githubhandle](https://github.com/HossainAraf)
-- Twitter: [@gmail](email2faizm@gmail.com)
+- GitHub: [@githubhandle](https://github.com/the-faizmohammad)
+- Twitter: [@twitter](https://twitter.com/FaizMoh58742600)
```

Comment:
Just as requested by the previous reviewer, kindly note that your twitter link is still broken and this can lead to a frustrating user experience. kindly do well to fix it.
<img width="804" alt="image" src="https://github.com/the-faizmohammad/School-Library-OOPs/assets/64643995/1a6c4bb4-9d94-45cb-8bdc-280f7dc06a76">